packages/ifpack/src/euclid/mat_dh_private.c

```diff
@@ -1166,7 +1167,7 @@ partition_and_distribute_metis_private (Mat_dh A, Mat_dh * Bout)
   /* broadcast number of rows to all processors */
   if (myid_dh == 0)
     m = A->m;
-  MPI_Bcast (&m, 1, MPI_INT, 0, MPI_COMM_WORLD);
+  MPI_Bcast (&m, 1, MPI_INT, 0, ifpack_get_global_comm());
```

Comment:
Both `partition_and_distribute_private` and `partition_and_distribute_metis_private` are only being used within `readMat_par` function which has the following [documentation](https://github.com/trilinos/Trilinos/blob/5aaae1ada6fe1ce777e671a0ff84fdc4f0779406/packages/ifpack/src/euclid/mat_dh_private.h#L87-L92):
```c++
/* Next function is primarily (?) for testing/development/debugging.
   P_0 reads and partitions the matrix, then distributes 
   amongst the other processors.
*/
  extern void readMat_par (Mat_dh * Aout, char *fileType, char *fileName,
			   int ignore);
```

I don't see `readMat_par` being used anywhere in Trilinos